src/coreclr/System.Private.CoreLib/src/Microsoft/Win32/OAVariantLib.cs

```diff
@@ -78,12 +79,19 @@ internal static Variant ChangeType(Variant source, Type targetClass, short optio
             ArgumentNullException.ThrowIfNull(culture);
 
             Variant result = default;
-            ChangeTypeEx(ref result, ref source,
-                         culture.LCID,
-                         targetClass.TypeHandle.Value, GetCVTypeFromClass(targetClass), options);
+            ChangeType(
+                Unsafe.AsPointer<Variant>(ref result),
+                Unsafe.AsPointer<Variant>(ref source),
```

Comment:
```suggestion
                &result,
                &source,
```
Since this is legal now?